client/extensions/wp-super-cache/state/cache/test/reducer.js

```diff
@@ -188,7 +188,7 @@ describe( 'reducer', () => {
 				type: SERIALIZE,
 			} );
 
-			expect( state.testing ).to.eql( {} );
```

Comment:
Feel free to remove related "should not load persisted state" tests. This is the default now 👍

Reply:
I feel like I got rid of most of these in #13542 but maybe people keep writing them?